.github/workflows/e2e-tests.yml

```diff
@@ -67,6 +67,8 @@ jobs:
           -e APP_ENV=test \
           pocket-id/pocket-id:test
 
+          docker logs -f pocket-id-sqlite &> /tmp/backend.log &
```

Comment:
Chanes to this file allow getting logs from the backend container too, since playwright logs don't have all info sometimes